scarb/tests/e2e/new_and_init.rs

```diff
@@ -25,6 +25,39 @@ fn new_simple() {
     assert!(t.child("Scarb.toml").is_file());
     assert!(t.child("src/lib.cairo").is_file());
     assert!(t.child(".gitignore").is_file());
+    assert!(t.child(".git").is_dir());
+
+    let toml_manifest = TomlManifest::read_from_path(t.child("Scarb.toml").utf8_path()).unwrap();
+    assert_eq!(toml_manifest.package.unwrap().name.as_str(), "hello");
+
+    Scarb::quick_snapbox()
+        .arg("build")
+        .current_dir(&t)
+        .assert()
+        .success();
+
+    t.child("target/release/hello.sierra")
+        .assert(predicates::str::is_empty().not());
+}
+
+#[test]
+fn new_simple_without_vcs() {
```

Comment:
Hmm, building Cairo code is the biggest factor of test suite execution time currently, so I would welcome avoiding it whenever possible. I think this test could limit itself just to ensure there's no `.git` and `.gitignore` and assume `new_simple` does the rest of the job. Wdyt?

This is not blocking. I can do this myself after merging this. I'd love to get this released today :)

Reply:
https://github.com/software-mansion/scarb/commit/3421a411ff6554a6692113da47f22f5894d77933

Reply:
I agree.

Reply:
Just a random suggestion but do you think it's possible if we avoid building at all in all the tests under `new_and_init`? Just have something like `check_can_build( )` where it checks that the created/initialized package has everything needed to build successfully?

Reply:
This approach has a problem that it's easy to forget to update it in case something changes :(

Reply:
We might consider implementing `scarb check` though, but this requires investigating if skipping Sierra codegen speeds up anything

Reply:
#137 I don't know if I'll open this for external contributors though, seems to have a potential for invasive changes which we prefer to keep in-house